website/views.py

```diff
@@ -142,26 +143,29 @@ def index():
 
     user_id = get_current_user_id()
     if user_id:  # Logged in: return either landing page or user home page
-        all_institutions = (
-            Institution.objects.filter(
-                is_deleted=False,
-                nodes__is_public=True,
-                nodes__is_deleted=False,
-                nodes__type='osf.node'
+        if waffle.switch_is_active('ember_dashboard'):
+            return send_from_directory(ember_osf_web_dir, 'index.html')
+        else:
```

Comment:
No need for an else here, since you're returning in the if statement. This will save the indentation/blame history.